File: Python/Exercicio11.py
# Algoritmo que calcula o peso ideal de uma pessoa, com base em sua altura e sexo informados.

# Entrada
altura = float(input("Informe a sua altura: "))
sexo = input("Informe o seu sexo (M/F): ")

# Processamento
if sexo == "M" or sexo == "m":
    peso_ideal = (72.7 * altura) - 58
elif sexo == "F" or sexo == "f":
    peso_ideal = (62.1 * altura) - 44.7
else:
    peso_ideal = None
    print("Sexo não reconhecido. ")

# Saída
if peso_ideal is not None:
    print("Seu peso ideal seria: {0:.2f}".format(peso_ideal))
else:
    print("Não foi possível calcular o peso ideal devido a sexo não reconhecido.")

File: C/Exercicio11.c
//Algoritmo que calcula o peso ideal, baseado em sua altura e sexo.
#include <stdio.h>
#include <ctype.h> //Para a função tolower

int main() {
    //Variáveis
    float altura, peso_ideal;
    char sexo;

    //Entrada
    printf("Informe sua altura em metros: \n");
    scanf("%f", &altura);
    while (getchar() != '\n'); //Limpa o buffer do input

    printf("Informe seu sexo (m/f): \n");
    scanf("%c", &sexo);

    //Processamento
     if (tolower(sexo) == 'm') { //tolower transforma o input em letra minúscula
        peso_ideal = (72.7 * altura) - 58;
        //Saída
        printf("Seu peso ideal é %.2f \n", peso_ideal);
    } else if (tolower(sexo) == 'f') {
        peso_ideal = (62.1 * altura) - 44.7;
        //Saída
        printf("Seu peso ideal é %.2f \n", peso_ideal);
    } else {
        //Saída
        printf("Sexo não reconhecido. \n");
    }
    return 0;
}